### ISSUE TITLE
syncthing running as gdm user

### ISSUE THREAD

**Issue by subpop** [decathorpe/fedora-pkgs#11]
When I installed the F26 beta and installed syncthing, I notice every time I boot up, syncthing gets started for the `gdm` user. This seems to cause some serious CPU utilization; the only reason I noticed this was because my fans spin up regularly until I kill the syncthing processes.

No systemd unit for `syncthing@gdm` is enabled. And no user systemd unit exists for syncthing in /var/lib/gdm. I can't figure out how it's getting started. Have you noticed this?

```
# tree /var/lib/gdm
/var/lib/gdm
└── Sync
```

**Comment by decathorpe** [comment]
I was made aware of this issue via e-mail a few days ago. After some troubleshooting, I found that fedora's default systemd configuration is different for system and user services - whereas there is a preset to disable all installed system services by default (`/usr/lib/systemd/system-presets/99-disable-default.preset`), there's no such thing for user services.

This caused the user service to be enabled system-wide when I included the appropriate / necessary scriptlets in the syncthing 0.14.30 package. However, the 0.14.31 update (which hit the COPR repository today) contains a fix to prevent this in the future.

To disable the erroneously activated systemd user service for *all* users (including gdm, etc.), run:

```
systemctl --user --global disable syncthing.service
```

This should fix the problem.
